daemons/dss-gs-event-relay/requirements.txt

```diff
@@ -1 +1 @@
-boto3
```

Comment:
I think this would be ok, it seems that this is just publishing SNS topics, we should not see interference to the lambda requirements.txt or the layers. 

 i would rather we remove the boto3 http adapter and use https://3.python-requests.org/user/advanced/#transport-adapters

Reply:
There's an issue for doing exactly that: https://github.com/HumanCellAtlas/data-store/issues/2553